mw-webapp/src/dataAccessLogic/getWaysPreview.ts

```diff
@@ -1,6 +1,5 @@
 import {wayDTOToWayPreviewConverter} from "src/dataAccessLogic/DTOToPreviewConverter/wayDTOToWayPreviewConverter";
```

Comment:
the same comments as for getUsersPreview

Reply:
Done